b/better-ajv-errors/LICENSE

```diff
@@ -0,0 +1,13 @@
+Copyright 2018 Atlassian Pty Ltd
```

Comment:
this is the script license and not the package license, can just be copied over from the base folder of this repo

Reply:
Updated license file